lib/escalate.rb

```diff
@@ -29,7 +32,7 @@ class << self
     #
     # @param [Hash] context
     #   Any additional context to be tied to the escalation
-    def escalate(exception, location_message, logger, **context)
+    def escalate(exception, location_message, logger, context: {})
```

Comment:
Note that `context:` is now a keyword here, but I didn't change that it's passed as a third argument hash to the `on_escalate` block.